src/uproot/exceptions.py

```diff
@@ -78,14 +78,7 @@ def __str__(self):
             in_object = f"\nin object {self.object_path}"
 
         if self.cycle == "any":
-            return """not found: {} (with any cycle number){}{}{}{}""".format(
-                repr(self.key), because, with_keys, in_file, in_object
-            )
-        elif self.cycle is None:
-            return """not found: {}{}{}{}{}""".format(
-                repr(self.key), because, with_keys, in_file, in_object
-            )
-        else:
-            return """not found: {} with cycle {}{}{}{}{}""".format(
-                repr(self.key), self.cycle, because, with_keys, in_file, in_object
-            )
+            return f"not found: {self.key!r} (with any cycle number){because}{with_keys}{in_file}{in_object}"
+        if self.cycle is None:
+            return f"not found: {self.key!r}{because}{with_keys}{in_file}{in_object}"
+        return f"not found: {self.key!r} with cycle {self.cycle}{because}{with_keys}{in_file}{in_object}"
```

Comment:
The formatting improvements are very good, but I want to keep match-case logic in explicit if-elif-else chains. The result is the same because all code branches return, but I think this expresses the intent more clearly.

```suggestion
        elif self.cycle is None:
            return f"not found: {self.key!r}{because}{with_keys}{in_file}{in_object}"
        else:
            return f"not found: {self.key!r} with cycle {self.cycle}{because}{with_keys}{in_file}{in_object}"
```

Reply:
I am not fond of adding extra "elif"/"else" that serve no purpose other than to confuse the reader into thinking they are meaningful, adding extra indentation, and making a more complicated AST. It's also more visually apparent to me that the final fall through case (else) is special, and that all possible branches are covered. When these are nested, it's even more important. 

But I wasn't making this change, this just slipped in because I was already editing the lines.